.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -12,25 +12,30 @@ assignees: ''
   - [ ] Notify maintainers about updating changelog for in-flight PRs 
 - [ ] Optional: for hotfixes, create a release branch and cherry-pick, e.g. `release-<version>-hotfix`
   - [ ] Optional: for hotfixes, create a PR into main from the hotfix branch to see the CI checks pass
-- [ ] On the appropriate branch/commit, create a calver tag for the release candidate, format example: `24.4.0-RC2`
-  - [ ] git tag 24.4.0-RC2
-  - [ ] git push upstream 24.4.0-RC2
+- [ ] On the appropriate branch/commit, create a calver tag for the release candidate, format example: `24.4.0-RC1`
+  - [ ] git tag 24.4.0-RC1
+  - [ ] git push upstream 24.4.0-RC1
 - [ ] Sign-off with team; announce the tag in #besu-release in Discord
-  - [ ] Targeting this tag for the burn-in: https://github.com/hyperledger/besu/releases/tag/24.4.0-RC2
+  - [ ] Targeting this tag for the burn-in: https://github.com/hyperledger/besu/releases/tag/24.4.0-RC1
 - [ ] Consensys staff start burn-in using this tag
 - [ ] Seek sign off for burn-in
   - [ ] Pass? Go ahead and complete the release process
   - [ ] Fail? Put a message in #besu-release in Discord indicating the release will be aborted because it failed burn-in 
-- [ ] Using the same git sha, create a calver tag for the FULL RELEASE, example format `24.4.0`
-- [ ] Using the FULL RELEASE tag, create a release in github to trigger the workflows. Once published:
+- [ ] Optional: Perform a dry run with https://github.com/consensys/protocols-release-sandbox to test the workflows
+  - [ ] Sync fork
+  - [ ] git checkout <sha of 24.4.0-RC1>
+  - [ ] git tag 24.4.0; git push origin 24.4.0
+  - [ ] Manually run https://github.com/Consensys/protocols-release-sandbox/actions/workflows/draft-release.yml using `main` branch and `24.4.0` tag
+- [ ] Back on besu, using the same git sha as 24.4.0-RC1, create a calver tag for the FULL RELEASE, example format `24.4.0`
```

Comment:
maybe step it out here as above
  - [ ] git checkout <sha of 24.4.0-RC1>
  - [ ] git tag 24.4.0; git push origin 24.4.0

Reply:
to make it super obvious that yes you are doing the exact same thing again in the main besu repo

Reply:
done